configs/caas/cromwell.conf.ctmpl

```diff
@@ -158,6 +162,26 @@ system {
     jobs = 50
     per = 1 second
   }
+
+  # More info: https://docs.google.com/presentation/d/14VgNBDE8Don_oeOOPIWtJQC5OUrcpk3Vip5AQBBCSs4/edit
+  hog-safety {
+    # Have seen a comment referring briefly to about "4k ips per person"(???).
+    # Meanwhile, this config file sets "concurrent-job-limit = 75000".
+    # Based on the calculations in the presentation above: 75k / 4k ~= 19
+    # If you know any additional information about the IP limit, etc., please update this comment.
+    hog-factor = 19
```

Comment:
Did we check with greenworld that this is what they want? I would guess they actually want "equivalent to no hog factors"... which would be "1"